### PR TITLE
Fixes#195. Fix RawDatagramSocket tests

### PR DESCRIPTION
**Pull request by sgrekhov**
This PR fixes `RawDatagramSocket `. 
1) Removed tests for members inherited from `Stream`
2) UDP protocol doesn't guarantee anything (delivery, order,  data integrity), so tests changed to:
  - send some data
  - if something was received, then check that recieved is part of the sent data

### CODE REVIEW

File: LibTest/io/RawDatagramSocket/bind_A02_t08.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+///   bool reusePort = false,
+///   int ttl = 1}
+/// )
+/// Binds a socket to the given host and port.
+/// ...
+/// The `reuseAddress` should be set for all listeners that bind to the same
+/// address. Otherwise, it will fail with a [SocketException].
+///
+/// @description Checks that if `reuseAddress` is `true` two sockets may share
+/// the same address
```
Comment (chloestefantsova) [inline LibTest/io/RawDatagramSocket/bind_A02_t08.dart:19]: Nit: the test checks something different than what's written in the description.

File: LibTest/io/RawDatagramSocket/writeEventsEnable_A01_t02.dart [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion bool writeEventsEnabled
+/// Set or get, if the [RawDatagramSocket] should listen for
+/// [RawSocketEvent.write] events. Default is true. This is a one-shot listener,
+/// and writeEventsEnabled must be set to true again to receive another write
+/// event.
+///
+/// @description Checks that if writeEventsEnabled is false, then this even is
```
Comment (chloestefantsova) [inline LibTest/io/RawDatagramSocket/writeEventsEnable_A01_t02.dart:11]: Nit: "even" ==> "event"

**Review by chloestefantsova — APPROVED** [review]
LGTM with a couple of nits.

**Comment by sgrekhov** [comment]
@chloestefantsova thank you. I've updated descriptions

**Comment by chloestefantsova** [comment]
> @chloestefantsova thank you. I've updated descriptions

Thanks!

**Pull request merged**